liquid_node/backup.py

```diff
@@ -223,10 +237,10 @@ def is_index_available(es_client, name):
 
 
 @retry()
-def backup_collection_es(dest, name):
+def backup_collection_es(dest, name, es_url):
```

Comment:
I think it's best to rename this to `def backup_es(dest, name, es_url_suffix, es_alloc_id):`